### ISSUE TITLE
[BUG]网速变动不显示，一直是0KB/S

### ISSUE THREAD

**Issue by aoeiuv-liu** [orblazer/plasma-applet-resources-monitor#5]
![2021-09-14_21-45](https://user-images.githubusercontent.com/20061353/133269342-e2bcd972-4c1e-48e6-a58c-530fb966d4b4.png)
刚安装那段时间可以正常显示，后面不知道为啥不行了。

**Comment by orblazer** [comment]
Hello,
Please write in English.

For your issue you have configure the network interface in `Data` ?

**Comment by aoeiuv-liu** [comment]
> Hello,
> Please write in English.
> 
> For your issue you have configure the network interface in `Data` ?

Yes, the maximum upload speed and maximum download speed are both set to 10,240KB/S
